vm-setup/roles/firewall/defaults/main.yml

```diff
@@ -1,13 +1,17 @@
 use_firewalld: true
 baremetal_interface: baremetal
 provisioning_interface: provisioning
+cluster_provisioning_interface: ironicendpoint
 external_subnet_v4: "{{ lookup('env', 'EXTERNAL_SUBNET_V4') | default('192.168.111.0/24', true) }}"
+provisioning_subnet: "{{ lookup('env', 'PROVISIONING_NETWORK') | default('172.22.0.0/24', true) }}"
```

Comment:
Could you document `PROVISIONING_NETWORK` in `vars.md` while at it? Seems to be missing.